### ISSUE TITLE
Allow options runOnly to be a string

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#3003]
This would be neat:

```js
const res = await axe.run({ runOnly: 'color-contrast' }); // For a single rule
const res = await axe.run({ runOnly: 'experimental' }); // For a single tag
```

I think this would be a fairly simple thing to do too. A simple tweak to lib/core/audit.js:

```js
if (typeof options.runOnly === 'sting') {
  options.runOnly = [options.runOnly];
}
```

Just needs tests, docs and a tweak to axe.d.ts.

**Comment by padmavemulapati** [comment]
Validated with the latest develop branch code base,
seeing `runOnly` option with  `string` instead `string array`
![image](https://user-images.githubusercontent.com/47409579/124435984-2b38b600-dd93-11eb-9265-e1e0da762239.png)
